include/dali/operators.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2019-2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_OPERATORS_H_
+#define DALI_OPERATORS_H_
+
+#ifdef __cplusplus
+///@{
+namespace dali {
+/**
+ * @brief Functions to initialize operators in DALI
+ *
+ * You should call this function once per process.
```

Comment:
Calling multiple times won't do any harm.

Reply:
That's true, but calling `daliInitialize` multiple times is forbidden. That's why I've put it here - to be consistent. I guess I can remove this comment, and we can add it later, if needed

Reply:
Maybe we should change that.

Reply:
And do something like in case of `cuInitChecked`.